modules/blocks/verify.js

```diff
@@ -300,14 +301,14 @@ __private.verifyPayload = function(block, result) {
 		result.errors.push('Number of transactions exceeds maximum per block');
 	}
 
-	var totalAmount = 0;
-	var totalFee = 0;
-	var payloadHash = crypto.createHash('sha256');
-	var appliedTransactions = {};
+	let totalAmount = 0;
+	let totalFee = 0;
+	const payloadHash = crypto.createHash('sha256');
+	const appliedTransactions = {};
 
-	for (var i in block.transactions) {
-		var transaction = block.transactions[i];
-		var bytes;
+	for (const i in block.transactions) {
```

Comment:
`For-of` method is a more reliable way of looping through an Array in sequence.
FYI: https://bitsofco.de/for-in-vs-for-of/